Framework/PythonInterface/plugins/algorithms/WorkflowAlgorithms/CorelliPowderCalibrationCreate.py

```diff
@@ -0,0 +1,576 @@
+        self.declareProperty(StringArrayProperty('ComponentList', values=self._banks, direction=Direction.Input),
+                             doc='Comma separated list on banks to refine')
+        self.declareProperty(name='ComponentMaxTranslation', defaultValue=0.02,
+                             doc='Maximum translation of each component along either of the X, Y, Z axes (m)')
+        self.declareProperty(name='ComponentMaxRotation', defaultValue=3.0,
+                             doc='Maximum rotation of each component along either of the X, Y, Z axes (deg)')
+        [self.setPropertyGroup(name, 'AlignComponents') for name in property_names]
+
+    def PyExec(self):
+        temporary_workspaces = []
+        self.temp_ws = temp_workspace_generator(temporary_workspaces)  # name generator for temporary workpaces
+
+        prefix_output = self.getProperty('OutputWorkspacesPrefix').value
+        progress_percent_start, progress_percent_end, reports_count = 0.0, 0.01, 5
+        progress = Progress(self, progress_percent_start, progress_percent_end, reports_count)
+        input_workspace = self.getProperty('InputWorkspace').value
+        adjustment_diagnostics = list()  # list workspace names that analyze the orientation of the banks
+
+        # Create a grouping workspace whereby we group detectors by banks
+        grouping_workspace = self.temp_ws()  # a temporary name
+        CreateGroupingWorkspace(InputWorkspace=input_workspace,
+                                OutputWorkspace=grouping_workspace,
+                                GroupDetectorsBy='bank')
+
+        # Remove delayed emission time from the moderator
+        kwargs = dict(InputWorkspace=input_workspace, Emode='Elastic', OutputWorkspace=input_workspace)
+        self.run_algorithm('ModeratorTzero', 0, 0.02, soft_crash=True, **kwargs)
+        progress.report('ModeratorTzero has been applied')
+
+        r"""Test to see if the build error goes away
+        # Correct detector heights
+        tube_calibration_database = self.getProperty('TubeDatabaseDir').value
+        yesno = 'not '  # flags if the tube calibration has been applied to the workspace
+        if os.path.isdir(tube_calibration_database):
+            tube_calibration_table_name = self.temp_ws()
+            tube_calibration_table, tube_calibration_mask = load_calibration_set(input_workspace,
+                                                                                 tube_calibration_database,
+                                                                                 tube_calibration_table_name,
+                                                                                 self.temp_ws())
+            if tube_calibration_table is not None:
+                apply_calibration(input_workspace, tube_calibration_table)
+                yesno = ''
+        progress.report(f'Tube calibration has {yesno}been applied')
+        """
```

Comment:
Similar to before, I would recommend remove debugging related comments before merging it into master.